edx_reverification_block/xblock/reverification_block.py

```diff
@@ -55,3 +68,11 @@ def get_course_id(self):
         # This is not the real way course_ids should work, but this is a
         # temporary expediency for LMS integration
         return self.course_id if hasattr(self, "xmodule_runtime") else "edX/Enchantment_101/April_1"
+
+    def get_org(self):
+        """ Return the org
+
+        """
+        # This is not the real way getting the org should work, but this is a
+        # temporary expediency for LMS integration
+        return self.xmodule_runtime.course_id.org if hasattr((self, "xmodule_runtime")) else "edX ORG"
```

Comment:
Shouldn't this be `hasattr(self, "xmodule_runtime")`?  (Two arguments, not a single tuple argument)